cv32e20/tests/cfg/default.yaml

```diff
@@ -1,9 +1,13 @@
 name: default
 description: Default configuration for CV32E20 simulations
 compile_flags: 
+# Alternate reference model configuration using parameter overrides and tool control
+#  --override cpu/priv_version=20190608
 ovpsim: >
-# Alternate reference model configuration using parameter overrides
-#    --override cpu/misa_Extensions=0x001104
-#    --override cpu/marchid=4
-#    --override cpu/noinhibit_mask=0xFFFFFFF0
+  --override cpu/mvendorid=0
```

Comment:
The value of `mvendorid` for OpenHW cores should be 0x0000_0602.  See the [CV32E40P User Manual](https://docs.openhwgroup.org/projects/cv32e40p-user-manual/en/latest/control_status_registers.html#machine-vendor-id-mvendorid) for an example.

Reply:
The RTL currently sets the mvendorid register to 0x0.  This is an override so the test will pass (until the RTL is changed).  What the correct value for mvendorid, and marchid is currently an unresloved discussion on MM / github.
The csr specification file lists mvendorid=0x00000602, and that is what the reference model uses.